.buildkite/gen-pipeline.sh

```diff
@@ -361,6 +361,10 @@ run_spark_integration() {
     run_test "${test}" "${queue}" \
       ":spark: Spark Torch MNIST (${test})" \
       "bash -c \"OMP_NUM_THREADS=1 python /horovod/examples/spark/pytorch/pytorch_spark_mnist.py --num-proc 2 --work-dir /work --data-dir /data --epochs 3\""
+
+    run_test "${test}" "${queue}" \
+      ":spark: Spark Torch MNIST (${test})" \
```

Comment:
the test title should be unique, please add 'Lightning' it.

Reply:
name changed.